backend/analytics_server/mhq/api/deployment_analytics.py

```diff
@@ -62,14 +60,10 @@ def get_team_deployment_analytics(
     pr_filter: PRFilter = apply_pr_filter(
         pr_filter, EntityType.TEAM, team_id, [SettingType.EXCLUDED_PRS_SETTING]
     )
-    code_repo_service = CodeRepoService()
 
-    team_repos: List[TeamRepos] = code_repo_service.get_active_team_repos_by_team_id(
-        team_id
-    )
-    org_repos: List[OrgRepo] = code_repo_service.get_active_org_repos_by_ids(
-        [str(team_repo.org_repo_id) for team_repo in team_repos]
-    )
+    pr_analytics_service = get_pr_analytics_service()
+
+    org_repos: List[OrgRepo] = pr_analytics_service.get_repo_by_id(team_id)
```

Comment:
@Kamlesh72 we have to fetch repos by team id but the function you used fetched repo by repo id.

Reply:
Also, you would have easily caught this if the code was tested once in any way.